packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -79,7 +80,8 @@ class BoxDecoration extends Decoration {
     this.borderRadius,
     this.boxShadow,
     this.gradient,
-    this.shape = BoxShape.rectangle,
+    this.backgroundBlendMode: BlendMode.srcOver,
```

Comment:
@Hixie might have more opinions. I'd just let this be nullable and let the default come from Paint so we don't add cross-package defaulting and to avoid the setter for every decoration construction.